descriptions/Engine.Solar2D.md

```diff
@@ -0,0 +1 @@
+[Solar2D](https://solar2d.com) is a cross-platform framework which is ideal for rapidly creating apps and games for mobile devices, TV, desktop systems and HTML5. That means you can create your project once and publish it to multiple devices, including Apple iPhone and iPad, Android phones and tablets, Amazon Fire, Mac Desktop, Windows Desktop, Linux, HTML5 and even connected TVs such as Apple TV, Amazon Fire TV, and Android TV.
```

Comment:
Remove the second sentence.

And I think it can be reworded a bit to say it is Lua based.

Reply:
Makes sense! Just updated it.